File: app/src/main/java/com/tmdb/codechallenge/home/HomeActivity.java
package com.tmdb.codechallenge.home;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.tmdb.codechallenge.R;
import com.tmdb.codechallenge.home.ui.HomeFragment;

public class HomeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.home_activity);

        getSupportFragmentManager().beginTransaction()
                .replace(R.id.container, new HomeFragment())
                .commit();
    }
}

File: app/src/main/java/com/tmdb/codechallenge/home/HomeActivity.kt
package com.tmdb.codechallenge.home

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.tmdb.codechallenge.R
import com.tmdb.codechallenge.home.ui.HomeFragment

class HomeActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.home_activity)

        supportFragmentManager.beginTransaction()
                .replace(R.id.container, HomeFragment())
                .commit()
    }
}
